File: 프로그래머스/lv2/12945. 피보나치 수/피보나치 수.js
function solution(n) {
    var answer = 0;
    if (n === 1) return 1;
    if (n === 2) return 1;
    let arr = [1, 1];
    for (let i = 2; i < n; i++) {
        arr.push((arr[i - 1] + arr[i - 2]) % 1234567);
    }
    answer = arr[n - 1];
    return answer;
}

File: 프로그래머스/lv2/12945. 피보나치 수/피보나치 수.py
def solution(n):
    answer = 0
    if n == 1:
        return 1
    if n == 2:
        return 1
    arr = [1, 1]
    for i in range(2, n):
        arr.append(arr[i - 1] + arr[i - 2])
    answer = arr[n - 1]
    return answer % 1234567